df.ui-menus.xml

```diff
@@ -421,9 +439,9 @@
 
             <stl-string name='worldgen_param_set'/>
             <int8_t name="unk_17d0"/>
-            <long name="num_speech_tokens"/>
-            <int8_t name="unk_17d8"/>
         </compound>
+        <int32_t name="num_speech_tokens"/>
```

Comment:
I'm fairly certain `long` was determined to be right earlier. Are you sure it was changed?

Reply:
Ida shows it is reading dword but offset matches long. To fix size and offset (and offset of following byte) I changed type and moved it outside the command_line.

Reply:
Ok, makes sense. We probably went with `long` to get the field after it to line up, but padding around structs could cause that alignment too.